control-plane/subcommand/server-acl-init/rules.go

```diff
@@ -154,6 +154,9 @@ namespace_prefix "" {
     policy = "write"
     intentions = "write"
   }
+  node_prefix "" {
+    policy = "read"
+  }
 {{- if .EnableNamespaces }}
```

Comment:
Just for posterity sake, this enables [this particular api call](https://github.com/hashicorp/consul-api-gateway/blob/4c6ee19207474ea318a96bf96bce6cbe5b8248b4/internal/k8s/service/resolver.go#L288). Without the ACL, the call returns an empty node set and the controller is unable to resolve any Consul services needed in an HTTPRoute configuration.